persistence/sql/index_nugets_sqlpersistence_[4,5).partial.md

```diff
@@ -0,0 +1,14 @@
+The SQL Persister requires the [**NServiceBus.Persistence.Sql**](https://www.nuget.org/packages/NServiceBus.Persistence.Sql) package, which contains:
+
+ * APIs for manipulating `EndpointConfiguration`.
+ * Runtime implementations of saga, timeouts, subscriptions and outbox persisters.
+ * Attributes used to define compile time configuration settings.
+ * **[Version 4.3+]** An MSBuild target that generates the required SQL installation scripts at compile time.
+ * Optionally runs SQL installation scripts at endpoint startup for development purposes. See [Installer Workflow](installer-workflow.md).
+
+WARNING: In Version 4.2 and below, The [NServiceBus.Persistence.Sql.MsBuild](https://www.nuget.org/packages/NServiceBus.Persistence.Sql.MsBuild) package must also be included to any project that requires generating SQL schema creation scripts. This includes any project that contains saga classes or hosts an NServiceBus endpoint.
+
+### Other packages
+
+* [NServiceBus.Persistence.Sql.MsBuild](https://www.nuget.org/packages/NServiceBus.Persistence.Sql.MsBuild) - In Version 4.2 and below, this package controlled script generation apart from the main package. In Version 4.3 and above this package is deprecated, as the generation capability is built into the main package.
```

Comment:
> In Version 4.2 and below, this package controlled script generation apart from the main package.

In Version 4.2 and below, this package must also be referenced to generate scripts


> In Version 4.3 and above this package is deprecated, as the generation capability is built into the main package.

Needs a comma after "above"